packages/std/src/math/uint128.rs

```diff
@@ -321,6 +321,7 @@ impl_mul_fraction!(Uint128);
 // of the conflict with `TryFrom<&str>` as described here
 // https://stackoverflow.com/questions/63136970/how-do-i-work-around-the-upstream-crates-may-add-a-new-impl-of-trait-error
 
+// uint to Uint
 impl From<Uint64> for Uint128 {
     fn from(val: Uint64) -> Self {
         val.u64().into()
```

Comment:
Shouldn't the code below this impl also use the new `primitive_to_wrapped_int` macro?

Reply:
This is Uint64 <-> Uint128, so no primitive here. I think if we want to use macros for those it would be better to have a separate macro

Reply:
Nevermind, you are completely right. I was confused with too much code here. Will adapt

Reply:
Done in last commit